tests/markdown.test.ts

```diff
@@ -17,15 +17,15 @@ describe("Markdown Rendering", () => {
   });
 
   test("Pages Rendered", () => {
-    mdFiles.forEach((file) => {
+    for (const file of mdFiles) {
```

Comment:
Please write some notes in the commit message as to why this is the best practice. The `for` loops feel like 6 of one, half a dozen of the other to me.